Packs/NutanixHypervisor/Integrations/NutanixHypervisor/NutanixHypervisor.yml

```diff
@@ -0,0 +1,880 @@
+category: Utilities
+commonfields:
+  id: Nutanix Hypervisor
+  version: -1
+configuration:
+- display: Server URL
+  additionalinfo: (e.g., https://example.net)
```

Comment:
replace with actual example

Reply:
@Itay4 you want me to use our Nutanix instance url?